doc/uftrace-report.md

```diff
@@ -58,6 +58,9 @@ REPORT OPTIONS
     `--data` option, index 1 is for data given by the `--diff` option, and index
     2 is for (percentage) differences between the two data.
 
+\--format=*TYPE*
+:   Show format style output. Currently, only csv style is supported.
+
```

Comment:
Please move this into the first commit.  Also I think we need a name for the default mode as well: how about "simple"?

Reply:
It looks good. 
Does the user need to input `--format=simple`? 
Or should it work with only `--format`?

Reply:
It should be `--format=simple` (or `--format simple`) since we don't support optional arguments.

Reply:
@namhyung Squash as the first commit and I modified the phrase a bit.